### PR TITLE
Candidate Release 0.8.0

### LINKED ISSUE #1096: Minimize Tendermint Info/Error Logs From Pex, P2P to allow more readability
**Is your feature request related to a problem? Please describe.**
Modify tendermint to turn PEX/P2P to debug logs or at the very least, change the default config to not show those

### LINKED ISSUE #1103: Remove Block Rollback from CLI
**Describe the bug**
When a user syncs their pocket node to the current state, it will crash and return am error every time i resume syncing my node(see example below):

```
error during handshake: error on replay: Wrong Block.Header.AppHash.  Expected 2ABDB006B6AE7286148AF213E34D2F598E91B4A4B9F361D73C7AF79644676689, got 1AD12ED32AEB480B14E93E19D23393EDB68190D1AE867234621923CEC5AC19A6
```

Even after the user updates their node to the **RC-0.5.1 nightly**, and conducts an unsafe rollback using the following command: `pocket util unsafe-rollback <random block number>

It will get caught up and return the same error(see above) at the same height it originally crashed on. 

**To Reproduce**
1. start a pocket node from 0 
2. if/when it crashes, start it again to resume operations 
3. wait for a few momemts
4. See an error

**Expected behavior**
the Node should resume syncing 

**Enviroment**
* Ubuntu LTS
* config.json DB = goleveldb

### LINKED ISSUE #1194: Remove Session DB
According the docs " pocket reset "
    Reset the Pocket node.
    Deletes the following files / folders:
        .pocket/data
        priv_val_key
        priv_val_state
        node_keys
- - - 
Is there any reason that session.db needs to survive a reset?
It seems to cause trouble.
Does it add value in any way by persisting?

### LINKED ISSUE #1204: [FEATURE REQUEST] Support for Operator Address
Modify pocket stake/unstake process to allow for optional unstaking/payment address (different from stake address)

Abstract: Pocket Network’s security model for Validator nodes could be greatly enhanced by a protocol and associated code modification which allowed an unstaking/payment address to be defined at the same time that the stake command is executed.  This backward compatible enhancement would allow all node runners to protect themselves in the event a bad actor gaining control of their server and accessing the private key which controls the staking address to unstake and eventually steal the stake.  This would also allow node owners to trustlessly delegate the operation of their nodes to third parties who would not be able to steal the funds.

Implementation:

As protocol modifications go... this is not excessive and should be fully backward compatible (although not retroactive).  Three modifications are required:

1.) Modify the actions associated with the stake command to accept an additional address - the unstake/payment address.
2.) Modify the coin minting action to pay the minted amount to the unstake/payment address (if one exists, otherwise pay as usual)
3.) Modify the actions associated with the unstake command to move the funds to the unstake/payment address (if one exists, otherwise pay as usual).  And to require NOT the validator's signature, but the unstake address' signature.

I’m not trying to minimize the amount of work, testing and auditing that a change like this would require, but I am saying that this (or something similar) would represent a huge increase in the day to day security of all nodes and I ask that it be given full consideration for inclusion in the road map of the 6.0 release.

Comments and questions and criticisms are welcomed.

### LINKED ISSUE #1256: Better bug reporting for misconfigured datadirectory
**Is your feature request related to a problem? Please describe.**
If the datadirectory (or contents) is misconfigured in anyway 'pocket start' will fail with no reporting logs

### LINKED ISSUE #1316: Wrong /v1/query/apps docs
Into /query/apps the rpc doc say that will returns `tokens` property but instead it returns `staked_tokens`
```json
{
  "address": "07a6c996fd7038171e2ccb11a5a7be1188f2bb6e",
  "public_key": "8680faa1b8c324249cebd024e225f348eccd0fe9d877531cc8a22c16767f8893",
  "jailed": false,
  "chains": [
    "0021"
  ],
  "max_relays": "83500",
  "status": 2,
  "staked_tokens": "50000000000",
  "unstaking_time": "0001-01-01T00:00:00Z"
}
```
https://github.com/pokt-network/pocket-core/blob/f794bc74b167bc870b5f169a32ac6d43c98b928d/doc/rpc-spec.yaml#L1337

### LINKED ISSUE #1320: Add to blocktxs query a way to determine parseable elements
**Is your feature request related to a problem? Please describe.**
Right now when you query blocktxs it returns
```
{
  "total_count": <number>,
  "txs":[]
}
```
The `total_count` is just the same of `txs.length` or `per_page` sent, this does not allow to know how much txs this block has without ask for another page and see if the response is 0 elements.

**Describe the solution you'd like**
The ideal should be like return `{total_txs: <total of txs for that block>, total_count: <like now>, txs: [] }` so this will allow to avoid an extra request to get an empty response.

**Describe alternatives you've considered**
Another way to do this is like `/query/apps` or `/query/nodes` returning the amount of pages based on the per_page sent.

### LINKED ISSUE #1345: [FEATURE REQUEST]  Live Reload chains.json
Now that nodes are becoming more and more multi-chain, the consequences of each restart are expanding and the odds of needing to modify the chains.json are expanding with it.

Hot swapping / reloading the chains.json doesn't seem (at least on the surface) to be a major project and it does not have a lot of risk either.

Thanks
Shawn

### LINKED ISSUE #1362: [FEATURE REQUEST] Enforce warning if node staking done with minimum allowed amount (15k POKT)
**Is your feature request related to a problem? Please describe.**
CLI 'pocket nodes stake' command accept without warning, if by typo/mistake/misconfiguration/unaware of outcome/lack of knowledge at the moment,  lowest possible amount of 15K POKT if provided. if this is done, will result in almost 100% certainty that staked amount will be burned.

**Describe the solution you'd like**
- Warning indicating high risks of using lowest possible amount. This is present in documentation but a more clear warning/description  may help avoid worst possible situation if enforce at CLI execution. 
- If lowest possible amount used, request at least 1-5  more POKTs with message describing reason for it to give time to node runner/stake entity to properly take corrective actions.

Thanks Vinicio,

### LINKED ISSUE #1368: Invalid evidence: evidence was already committed
Occasionally, proposals blocks are rejected with the following evidence issue.

**Describe the bug**
```
2021-11-03T17:59:33.001508547Z stdout F E[2021-11-03|13:59:33.001] enterPrevote: ProposalBlock is invalid
module=consensus height=41729 round=0 err="Invalid evidence: evidence was already committed. Evidence: VoteA: 
Vote{1803:802B22B94DF4 25948/00/1(Prevote) 8661755EF3C7 D19329204A6A @ 2021-05-14T02:59:49.764895391Z}; VoteB:
Vote{1803:802B22B94DF4 25948/00/1(Prevote) 000000000000 9B5CA3FD9C4A @ 2021-05-14T02:59:43.094160623Z}" 
proposer=72967B79F47A3780C139E7A85BCAF00880D80DE4
```

**Expected behavior**
Tendermint evidence committing should be deterministic. 

**Additional context**
This issue is seen in RC-0.6.4

### LINKED ISSUE #1369: Remove Force-Unstake *burn* functionality
**Is your feature request related to a problem? Please describe.**
Force Unstake burns all stake instead of refunding... It has done nothing except disproportionally burn for falling below the minimum stake. 21 waiting period is enough of a punishment. I propose to simply refund the POKT instead of the force burn.

### LINKED ISSUE #1372: ReplayAttackError not implemented
**Describe the bug**
Replay attacks are stopped in RC-0.6.4 but the handler is not triggered through the validate

x/pocketcore/keeper/handle.go

validateProof()

```
if err.Code() == types.CodeReplayAttackError && !claim.IsEmpty() { // never called
			// delete local evidence
			processSelf(ctx, k, proof.GetSigner(), claim.SessionHeader, claim.EvidenceType, sdk.ZeroInt())
			// if is a replay attack, handle accordingly
			k.HandleReplayAttack(ctx, addr, sdk.NewInt(claim.TotalProofs))
			err := k.DeleteClaim(ctx, addr, claim.SessionHeader, claim.EvidenceType)
			if err != nil {
				ctx.Logger().Error("Could not delete claim from world state after replay attack detected", "Address", claim.FromAddress)
			}
		}
```

Meaning the punishment isn't elevated to the level the protocol expects

### PR DESCRIPTION
**Pull request by oten91**
Closes #1194
Closes #1369
Closes #1362
Closes #1372
Closes #1103
Closes #1316
Closes #1320
Closes #1345
Closes #1256
Closes #1096
Closes #1368
Closes #1204

### CODE REVIEW

File: doc/guides/quickstart.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ git checkout tags/<release tag>
 Example:
 
 ```text
-git checkout tags/RC-0.6.3
```
Comment (iajrz) [inline doc/guides/quickstart.md:38]: This is still recommending 0.6.3.

File: app/cmd/cli/node.go [hunk 2]
```diff
@@ -43,13 +45,22 @@ If the node is currently staked at X and you submit an update with new stake Y.
 If no changes are desired for the parameter, just enter the current param value just as before`,
 	Args: cobra.ExactArgs(6),
 	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("### Warning ###")
+		fmt.Println("Using Legacy Command , this will not work after RC 0.8.0 upgrade is activated.")
```
Comment (iajrz) [inline app/cmd/cli/node.go]: Space before comma is a typo.

**Review by iajrz — DISMISSED** [review]
Save for a couple of minor typos and maybe some change that could be made to the way Cobra is being used -- none of which affect functionality -- this looks good to me.

File: app/cmd/cli/node.go (hunk 2 above)
Comment (oten91) [reply app/cmd/cli/node.go]: fixed

File: doc/guides/quickstart.md (hunk 1 above)
Comment (oten91) [reply doc/guides/quickstart.md:38]: fixed

**Review by iajrz — APPROVED** [review]

**Pull request merged**